src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -52,11 +53,31 @@ describe('lib/viewers/image/ImageViewer', () => {
         stubs = {};
     });
 
+    describe('destroy()', () => {
+        test('should remove the zoom event listener', () => {
+            image.options.enableAnnotationsImageDiscoverability = true;
+            image.removeListener = jest.fn();
+
+            image.destroy();
+
+            expect(image.removeListener).toBeCalledWith('zoom', expect.any(Function));
+        });
+    });
+
     describe('setup()', () => {
         test('should set up layout', () => {
             expect(image.wrapperEl).toHaveClass('bp-image');
             expect(image.imageEl).toHaveClass('bp-is-invisible');
         });
+
+        test('should bind zoom listener if enableAnnotationsImageDiscoverability is true', () => {
```

Comment:
Should there be a similar test for asserting that no zoom listener is added if the feature is false?